docs/user-guide/liveness/README.md

```diff
@@ -66,11 +66,13 @@ The [http-liveness.yaml](http-liveness.yaml) demonstrates the HTTP check.
       httpGet:
         path: /healthz
         port: 8080
+        httpHeaders:
+          X-Custom-Header: [ Awesome ]
```

Comment:
Yeah, that's a good question. @thockin's way is probably more consistent with the rest of the API but the current way is going to be a lot more compact. I don't think there's much risk of us wanting to add other fields in here, so I guess I have an extremely minor preference for the current way. up to @thockin